articles/support/index.md

```diff
@@ -126,13 +126,13 @@ Critical: Defined as an Auth0 issue severely impacting your live or in-productio
 
 Critical Production issues should always be reported via the [Support Center](${env.DOMAIN_URL_SUPPORT}) for fastest response.
 
-Translation from former support plans to new:
-
 - Subscriptions previously known as "Free" map to the current "Free" support classification
 - Subscriptions previously known as "Gold/Silver/Home Grown"  map to the current "Standard" support plan.
+- Customers that have Developer or Developer Pro subscriptions map to the current "Standard" support plan.
```

Comment:
Perhaps we should add a "Subscription Plan" row to the table instead of specifying these in here. The content of each cell would be the names of the subscription plans that correspond to that support level.